Sources/Commands/SwiftTool.swift

```diff
@@ -302,15 +283,18 @@ public class SwiftTool {
     ///
     /// - parameter args: The command line arguments to be passed to this tool.
     public init(options: SwiftToolOptions) throws {
+        // first, bootstrap the observability system
+        ObservabilitySystem.bootstrapGlobal(factory: SwiftToolObservability())
```

Comment:
Is this roughly how the initialization will look even after the DiagnosticEngine compatibility ships are removed?

Reply:
yes!